Source/LightboxController.swift

```diff
@@ -210,7 +210,7 @@ open class LightboxController: UIViewController {
       x: 0,
       y: 16,
       width: view.bounds.width,
-      height: 100
+      height: 45
```

Comment:
100 was way bigger than necessary but it didn't matter since the header just overlayed the content.  Now we use this header size to adjust the actual content size (and avoid overlapping), so using a more appropriate value.